<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{border-image:url(background-image/back_3.jpg)}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="languagebutton">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>106</y>
      <width>464</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="languagebutton_second">
    <property name="geometry">
     <rect>
      <x>758</x>
      <y>106</y>
      <width>464</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text_reception">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>190</y>
      <width>462</width>
      <height>529</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text_output">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>462</width>
      <height>529</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QToolButton" name="contact_google">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>770</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/google.png);
	icon-size: 10 px;
	border: None;
}

QToolButton:pressed{
  image: url(icons/selected.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>739</y>
      <width>1161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="contact_tg">
    <property name="geometry">
     <rect>
      <x>1155</x>
      <y>770</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/tg.png);
	icon-size: 10 px;
	border: None;
}
QToolButton:pressed{
  image: url(icons/selected.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="contact_vk">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>770</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/vk.png);
	icon-size: 10 px;
	border: None;
}
QToolButton:pressed{
  image: url(icons/selected.png);
}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="footer">
    <property name="geometry">
     <rect>
      <x>586</x>
      <y>750</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>© 2021–2021</string>
    </property>
   </widget>
   <widget class="QLabel" name="company_name">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>750</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Scantype Black PERSONAL USE</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string> k h a i r u l l o v</string>
    </property>
   </widget>
   <widget class="QToolButton" name="arrow_changes">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>95</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/arrow_changes.png);
	icon-size: 100 px;
	border: None;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="copy_paste">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>690</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/copy_paste.png);
	icon-size: 40 px;
	border: None;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="copy_paste_2">
    <property name="geometry">
     <rect>
      <x>1164</x>
      <y>690</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; dir='rtl' style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Копировать&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/copy_paste.png);
	icon-size: 40 px;
	border: None;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="translate_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/logo.png);
	icon-size: 10 px;
	border: None;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
